sdk/tests/conformance2/rendering/blitframebuffer-test.html

```diff
@@ -245,11 +256,81 @@
     gl.deleteFramebuffer(fb1);
 };
 
-if (!gl) {
-    testFailed("WebGL context does not exist");
-} else {
-    testPassed("WebGL context exists");
-    blit_framebuffer_test();
+function blit_framebuffer_multisampling_srgb() {
+
+    debug("");
+    debug("This test vefify the functionality of blitframebuffer from or to a multisampled srgb image.");
+
+    // Read buffer can have multisampled srgb image, but draw buffers can not.
+    var rb0 = gl.createRenderbuffer();
+    var fb0 = gl.createFramebuffer();
+    var rb1 = gl.createRenderbuffer();
+    var fb1 = gl.createFramebuffer();
+    gl.bindRenderbuffer(gl.RENDERBUFFER, rb0);
+    gl.renderbufferStorageMultisample(gl.RENDERBUFFER, 4, gl.SRGB8_ALPHA8, width, height);
```

Comment:
This really should query gl.MAX_SAMPLES properly, rather than assuming that 4 samples will work. Or are we already making this assumption throughout the conformance suite? + @zhenyao